deploy/bare-metal/nebula.json

```diff
@@ -0,0 +1,1078 @@
+{
```

Comment:
We can add a `grafana` folder and move this json file into it for easy understanding.

Reply:
this is done in #2, change centos:7 to busybox